source/Ship.cpp

```diff
@@ -2710,6 +2710,21 @@ int Ship::Scan(const PlayerInfo &player)
 	if(isYours || (target->isYours && activeScanning))
 		Audio::Play(Audio::Get("scan"), Position());
 
+	bool isImportant = false;
+	if(target->isYours)
+	{
+		bool hasIllegal = target->Cargo().IllegalCargoFine(government);
+		if(!hasIllegal)
+			for(auto &it : target->Outfits())
+				if(government->Fines(it.first) || government->Condemns(it.first))
+				{
+					hasIllegal = true;
+					break;
+				}
+
+		isImportant = hasIllegal || target.get() == player.Flagship();
+	}
+
```

Comment:
I feel like this stuff should be done somewhere else. In Government or maybe a new class.
Adding this here is another place that needs to be updated if the way fining works is changed, like what #8677 tries to do.

Reply:
done